src/pages/CustomerRequisitionPage.js

```diff
@@ -240,9 +243,17 @@ export const CustomerRequisition = ({
     return (
       <View style={verticalContainer}>
         <Buttons />
+        {program && (
+          <ToggleBar
+            toggles={[
```

Comment:
Just a toggle bar when there is a program related which will toggle the column set. The 'a' column set will be exaclty the same as currently, so just not showing this toggle will mean no behaviour change for normal requisitions. 

Haven't done localization as I'm not sure what to call these buttons!